src/tracer/ideal_age_example.F90

```diff
@@ -340,27 +374,122 @@ subroutine ideal_age_tracer_column_physics(h_old, h_new, ea, eb, fluxes, dt, G,
   year = US%s_to_T*time_type_to_real(CS%Time) * Isecs_per_year
 
   do m=1,CS%ntr
-    if (CS%sfc_growth_rate(m) == 0.0) then
-      sfc_val = CS%young_val(m)
+
+    if (CS%growth_rate(m) == 0.0) then
+      young_val = CS%young_val(m)
     else
-      sfc_val = CS%young_val(m) * &
-          exp((year-CS%tracer_start_year(m)) * CS%sfc_growth_rate(m))
+      young_val = CS%young_val(m) * &
+        exp((year-CS%tracer_start_year(m)) * CS%growth_rate(m))
```

Comment:
According to the MOM6 style guide at https://github.com/mom-ocean/MOM6/wiki/Code-style-guide, "Continuation lines should be indented at least 4 spaces, but more space can be used if it helps align lines".  Line 382 is a continuation line with a 2-point indent.  Please increase this to at least 4-spaces.